assets/templates/layouts.data.hbs

```diff
@@ -15,10 +15,14 @@
     <div class="timeline-col">
 
         <div class="listing timeline">
+            <div class="bulk-action">
+            <button id="bulk-action" class="button-link bulk-action-secondary">Bulk Actions</button>
+            <div class="bulk-action-primary">6 of 6 Reports</div>
+            </div>
 
```

Comment:
Please indent html. Also, shouldn't button be last in html order, since it is second in line?

Reply:
I was following how listing-item-primary and listing-item-secondary are done. It's pretty much the exact same thing with some padding changes.